src/PluginSuppression.php

```diff
@@ -491,4 +493,34 @@ private function is_callback_plugin_suppressed( $callback, $suppressed_plugins )
 			in_array( $source['name'], $suppressed_plugins, true )
 		);
 	}
+
+	/**
+	 * Add meta if plugin is suppressed in AMP page.
+	 *
+	 * @param array  $plugin_meta An array of the plugin's metadata.
+	 * @param string $plugin_file Path to the plugin file relative to the plugins directory.
+	 *
+	 * @return array An array of the plugin's metadata
+	 */
+	public function filter_plugin_row_meta( $plugin_meta, $plugin_file ) {
+
+		$screen = get_current_screen();
+
+		if ( ! ( $screen instanceof WP_Screen && 'plugins' === $screen->id ) ) {
```

Comment:
If so, another option would be:

```suggestion
		if ( is_network_admin() ) {
```